user_docs/en/userGuide.t2t

```diff
@@ -2840,6 +2840,13 @@ If the add-on was previously "disabled", the status will show "enabled after res
 If the add-on was previously "enabled", the status will show "disabled after restart".
 Just like when you install or remove add-ons, you need to restart NVDA in order for changes to take effect.
 
++++ Reviewing and consulting feedback for add-ons +++[AddonStoreReviews]
+Before installing an add-on, you may want to see opinions and reactions by others.
+Also, you may want to provide feedback about add-ons to help other users.
+To see or share your feedback about add-ons, select an add-on and use the See or provide feedback option in the actions menu.

```

Comment:
I really do think that "read" is a better word than "see" for both the option and the usage. Not for any "ableist language" reason, but just because it flows better in English IMO.

Also, for this one, you are not going to "see your feedback", or even "read your feedback"--you are going to either read others' feedback, or leave your own.

Lastly, I think the menu option should be shorter--perhaps just "Reviews". I have only changed the word "see" in the suggestion though.

```suggestion
To read what others have said, or share your feedback about add-ons, select an add-on and use the Read or provide feedback option in the actions menu.
```